app/views/submission_similarities/show.html.erb

```diff
@@ -39,12 +39,16 @@
       </tr>
     </thead>
     <tbody>
-      <% @submission_similarity.mappings.sort_by { |m| m.start_line1 }.each { |mapping| %>
+      <% @submission_similarity.similarity_mappings.sort_by { |m| m.start_line1 }.each { |mapping| %>
```

Comment:
Again, any problems if the number satisfying is very large?  Do we need a `limit` clause?

Reply:
Same as above. Plus, since the very beginning e.g. before this change, we had index on submission_similarity_id column in submission_similarity_mappings table, so the query won't cause performance issues.